pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -110,7 +111,7 @@ func (s *Service) getOrCreateLoadBalancer(openStackCluster *infrav1.OpenStackClu
 	}
 
 	if err := s.waitForLoadBalancerActive(lb.ID); err != nil {
-		record.Warnf(openStackCluster, "FailedCreateLoadBalancer", "Failed to create load balancer %s: %v", loadBalancerName, err)
+		record.Warnf(openStackCluster, "FailedWaitForLoadBalancerActive", "Failed to wait for load balancer active %s with id %s: %v", loadBalancerName, lb.ID, err)
```

Comment:
I think I would keep the event name the same and only change the description. Essentially `FailedCreateLoadBalancer` is still correct as that's done asynchronously in OpenStack. What do you think?

(also applies to the other changes)